### PR TITLE
Improve Masu AWS report ingest logging clarity

### LINKED ISSUE #1989: Adjust ingestion logging to improve clarity
**Describe the bug**

During ingestion, Masu logs these lines:

```
[2020-03-31 11:38:08,821] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a Downloading /koku/20200301-20200401/koku-Manifest.json to /var/tmp/masu/processing/acct6193296/aws/koku/koku-Manifest.json
[2020-03-31 11:38:08,959] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a Added 5f3767da-d63b-4833-b68c-bfb18489f070:2020-03-01 00:00:00 to cache.
[2020-03-31 11:38:08,959] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a Deleted manifest file at /var/tmp/masu/processing/acct6193296/aws/koku/koku-Manifest.json
[2020-03-31 11:38:08,959] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a Inserting manifest database record for assembly_id: 18888647-d4f4-46a2-ab6f-d534351d1439
[2020-03-31 11:38:08,962] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a No manifest entry found.  Adding for bill period start: 2020-03-01 00:00:00
[2020-03-31 11:38:08,990] INFO 5102cbcc-6e28-4392-9fe0-a939c3d67b5a Local S3 filename: 18888647-d4f4-46a2-ab6f-d534351d1439-koku-1.csv.gz
```

However, it can be confusing what our ingestion is really up to. Is Masu trying to insert a manifest it just got finished deleting?  That isn't what Masu is doing, but the log messages seem to say that.

**To Reproduce**
1. Ingest an AWS report
2. Watch the Worker logs

**Expected behavior**

We have a few options on how to fix this:
1. Switch some of these messages to `DEBUG`, because there is a low probability of that code section failing. (e.g. file deletes don't fail very often, so we don't need them to be in the console logs for every ingestion run.)
2. Update the log message to be clearer about what is happening. (e.g. make it explicit what is a filesystem operation and what is a database operation.)
3. We can adjust the ordering of the ingestion steps so that all deletes happen after ingestion. This will naturally reorder the log messages in a way that makes more sense. Also, this might be useful if we want the ability to reprocess the entire report without first re-downloading the report.

### PR DESCRIPTION
**Pull request by boaz0**
related #1989

### CODE REVIEW

File: koku/masu/processor/worker_cache.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def add_task_to_cache(self, task_key):
         task_list.append(task_key)
         self.set_host_specific_task_list(task_list)
         msg = f"Added {task_key} to cache."
-        LOG.info(msg)
+        LOG.debug(msg)
```
Comment (adberglund) [inline koku/masu/processor/worker_cache.py:76]: I think we want to always log this. We can make the statement more informative, but we need to see the cache is working as expected.

File: koku/masu/processor/worker_cache.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def add_task_to_cache(self, task_key):
         task_list = self.host_specific_worker_cache
         task_list.append(task_key)
         self.set_host_specific_task_list(task_list)
-        msg = f"Added {task_key} to cache."
+        msg = f"Added task key {task_key} to Redis cache."
```
Comment (adberglund) [inline koku/masu/processor/worker_cache.py:75]: We actually just switched away from using Redis to caching this in the DB.

Comment (boaz0) [reply koku/masu/processor/worker_cache.py:75]: Oops, didn't know. Let me fix that... :seedling:

**Comment by boaz0** [comment]
ping @adberglund - I updated the commit can you take a look please :pray: 
Thanks :smile:

**Comment by boaz0** [comment]
Thanks @blentz & @adberglund !
Can I merge now?

**Comment by adberglund** [comment]
@boaz0 Yes, go ahead.

**Comment by boaz0** [comment]
Thanks you all!
@adberglund @blentz :sparkling_heart:

**Pull request merged**